.travis.yml

```diff
@@ -2,28 +2,19 @@ language: cpp
 compiler:
     - clang
     - g++
-before_install:
-    - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-    - sudo apt-get update -qq
-    - wget http://www.cmake.org/files/v3.1/cmake-3.1.2-Linux-x86_64.sh
-    - chmod +x cmake-3.1.2-Linux-x86_64.sh
-    - mkdir cmake
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+      - llvm-toolchain-precise
+    packages:
+      - gcc-4.9
+      - g++-4.9
+      - clang-3.7
 install:
-    - sudo apt-get install -y -qq libcunit1 libcunit1-dev libglfw-dev libglew-dev libfreeimage-dev xorg-dev gcc-4.9 g++-4.9
+    - sudo apt-get install -qq xorg-dev libcunit1-dev libglew-dev libglfw-dev libfreeimage-dev
```

Comment:
Did we want to try and nail down specific dependencies instead of just `xorg-dev`?